src/commands/soundboard/sb/index.ts

```diff
@@ -43,7 +43,7 @@ class Soundboard extends Command {
             readable.destroy();
         }
 
-        return message.react("👍");
+        if (!message.deleted) message.react("👍");
```

Comment:
To keep the last action of the method being the 👍, it might be better to early return instead of the message is deleted like so : 

```typescript
if (!message.deleted) { return; }

return message.react(...);
```